templates/module/service-template.ts

```diff
@@ -0,0 +1,37 @@
+import $entityNameReader from './internals/$moduleName-reader';
+import $entityNameWriter from './internals/$moduleName-writer';
+
+import { $entityName, Get$entityNameParams, PaginatedResponse } from './types';
+
+export default class $moduleNameService {
+  public static get$entityNames({
+    page,
+    size,
+  }: Get$entityNameParams): Promise<PaginatedResponse<$entityName>> {
+    return $entityNameReader.get$entityNames({
+      page: page || 1,
+      size: size || $pageLimit,
```

Comment:
Can we pass the default values from the controller itself. Also can we rename `$pageLimit` to `$defaultPageSize` or similar.